_inc/client/plans/single-product-search/index.jsx

```diff
@@ -2,21 +2,53 @@
  * External dependencies
  */
 import React from 'react';
-import { translate as __ } from 'i18n-calypso';
+import { translate as __, numberFormat } from 'i18n-calypso';
 import { get, noop } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import PlanRadioButton from '../single-product-components/plan-radio-button';
+import {
+	JETPACK_SEARCH_TIER_UP_TO_100_RECORDS,
+	JETPACK_SEARCH_TIER_UP_TO_1K_RECORDS,
+	JETPACK_SEARCH_TIER_UP_TO_10K_RECORDS,
+	JETPACK_SEARCH_TIER_UP_TO_100K_RECORDS,
+	JETPACK_SEARCH_TIER_UP_TO_1M_RECORDS,
+	JETPACK_SEARCH_TIER_MORE_THAN_1M_RECORDS,
+} from '../../lib/plans/constants';
 
-const PLACEHOLDER_RECORD_COUNT = 53;
-const PLACEHOLDER_PRICE = 50;
-const PLACEHOLDER_LABEL = 'Tier 1: Up to 100 records';
+function getTierLabel( priceTierSlug, recordCount ) {
+	switch ( priceTierSlug ) {
+		case JETPACK_SEARCH_TIER_UP_TO_100_RECORDS:
+			return __( 'Tier 1: Up to 100 records' );
```

Comment:
Let's remove the numbering for the tiers here and just say the number of records. We may add new tiers at some point.

Reply:
👍